roles/openshift_common/tasks/atomic_docker_check.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Fail early if Atomic and old version of Docker
+- block:
+
+  - name: Determine available Docker
+    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\n' docker"
```

Comment:
Thanks for the suggestion on advanced syntax. Unfortunately it doesn't work for commands ... even when setting via ``vars``. The result ends up being parsed to ``{#json .Server#}``.